libs/markets/src/lib/components/last-24h-price-change/last-24h-price-change.tsx

```diff
@@ -18,12 +18,15 @@ interface Props {
   initialValue?: string[];
   isHeader?: boolean;
   noUpdate?: boolean;
+  // to render nothing instead of '-' when there is no price change
+  hideZero?: boolean;
 }
 
 export const Last24hPriceChange = ({
   marketId,
   decimalPlaces,
   initialValue,
+  hideZero,
```

Comment:
This would be better as a render prop so this component is more flexible/declaritive

Reply:
updated